File: benchmarks/array-100K/main.go
package main

import "fmt"
import "time"

func main() {
	var before = time.Now().UnixNano()

	var list = make([]int, 100_000)
	for i := 0; i < 100_000; i++ {
		list[i] = i
	}

	var after = time.Now().UnixNano()

	fmt.Println(after - before)
}

File: benchmarks/array-100K/main.rs
use std::time::{SystemTime};

fn main() {
    let before = SystemTime::now().duration_since(SystemTime::UNIX_EPOCH).unwrap();

    let mut arr = [0; 100_000];

    for i in 0..100_000 {
        arr[i] = i;
    }

    let after = SystemTime::now().duration_since(SystemTime::UNIX_EPOCH).unwrap();
    println!("{}", after.as_nanos() - before.as_nanos());
}